scrapy/signalmanager.py

```diff
@@ -1,12 +1,17 @@
 from __future__ import absolute_import
-from pydispatch import dispatcher
+import warnings
+
 from scrapy.utils import signal as _signal
+from scrapy.exceptions import ScrapyDeprecationWarning
+from scrapy.dispatch.robustapply import robustApply
+from scrapy.dispatch.utils import func_accepts_kwargs
 
 
 class SignalManager(object):
 
-    def __init__(self, sender=dispatcher.Anonymous):
+    def __init__(self, sender=None):
         self.sender = sender
+        self.signals = {}
```

Comment:
Why is this dictionary called `signals` when it actually contains (patched) receivers?